t/lib/OpenQA/Test/Utils.pm

```diff
@@ -187,17 +188,23 @@ sub create_websocket_server {
     if ($wspid == 0) {
         $ENV{MOJO_LISTEN}             = "http://127.0.0.1:$port";
         $ENV{MOJO_INACTIVITY_TIMEOUT} = 9999;
+
         use OpenQA::WebSockets;
         use Mojo::Util 'monkey_patch';
         use OpenQA::WebSockets::Server;
+        use OpenQA::WebSockets::Controller::Worker;
+        use OpenQA::WebSockets::Plugin::Helpers;
+
+        # TODO: Kill it with fire!
```

Comment:
Can you perhaps add a hint as to what you'd like to see here? You've mentioned it before in IRC but such comments quickly lose context

Reply:
Unfortunately i don't have a solution for this specific case yet. It is different from the other uses of mocked functions/methods, because it hidden deep inside a test module and there is no indication of what is happening from the API side. So there's some very dangerous action at a distance stuff going on here.